src/execution/interpreter_loop.rs

```diff
@@ -33,19 +35,22 @@ use crate::execution::hooks::HookSet;
 
 /// Interprets a functions. Parameters and return values are passed on the stack.
 pub(super) fn run<H: HookSet>(
-    wasm_bytecode: &[u8],
-    types: &[FuncType],
-    store: &mut Store,
+    modules: &mut [ExecutionInfo],
+    current_module: &mut usize,
```

Comment:
Should we rename `current_module` here to `current_module_idx` and take `modules[*current_module]` outside of each match arm and put it before the loop so it's cleaner?
What do you guys think?